### DISCUSSION TITLE
Recommend RGBWWCW GU10 Bulb

### DISCUSSION THREAD

**Discussion by mode2k** [Koenkk/zigbee2mqtt#16396] [Devices]
Hallo,
what is the currently best supported GU10 RGBWWCW bulb to use with zigbee2mqtt?
I tried one Lidl [HG07834A](https://www.zigbee2mqtt.io/devices/HG07834A.html#lidl-hg07834a) witch is cheap and has a low stby consumption but power_on_behavior can not be set but this a must have for me.

What other brands/types of bulb can you recommend? Is ikea tradfi a good one?

br

mode

**Comment by yurgh** [comment]
IKEA TRADFRI bulb GU10 CWS 345lm (LED1923R5) has power-on behaviour:
![2023-01-28](https://user-images.githubusercontent.com/17332440/215272016-43128c05-70ce-46d5-80a2-5d534754eb4f.jpg)

**Reply by mode2k** [reply]
Is this bulb stable an durable? I will try it....
